back/engines/commercial/verification/app/services/verification/verification_service.rb

```diff
@@ -37,6 +37,13 @@ def active?(configuration, method_name)
       active_methods(configuration).include? method_by_name(method_name)
     end
 
+    # Not all verification methods are allowed at a permission/action level
+    def first_method_enabled_on_action
```

Comment:
So this method already exists in the BE, but as far as I can see I can't use this to get the first enabled method directly on the FE, right?

I guess I can get the all the `/verification_methods`, on the FE, and then replicate this logic there. I would need to do that definitely in the back office interface. Then for the flow in the front office, I would need either also this behavior, or (if we will show multiple SSO methods on staging platforms when verified action is chosen, which might not make sense if the back office only allow configuring one) go through all methods and them filter them by if they are supported or not.

So maybe we should have a new endpoint for the FE to directly get the first verified action-supported verification method?

Reply:
Yes good idea, I did think about that originally, but then meant to ask you! So I will add another endpoint with the same structure but for the first enabled method